README.md

```diff
@@ -88,6 +88,21 @@ FrederickAPI::V2::Location.with_access_token(access_token) do
 end
 ```
 
+### Using Headers
+
+`JsonApiClient::Resource` exposes a way to add headers that is written over in the `with_access_token` method.
+An example use case is adding an `X-Universal-Customer` header.
+
+```ruby
```

Comment:
This is an example of how you'd use `with_access_token_and_headers`, from this gem, in your app to make an API call with headers?

Reply:
Correct